ncm-ceph/src/main/perl/Ceph/commands.pm

```diff
@@ -161,8 +161,8 @@ sub ssh_known_keys {
         if (!$lines) {
```

Comment:
($key_accept eq 'first') means: only add the ssh key blindy if it is not already there. If it is there, it will not replace it, it only does that when key_accept is 'always'